packages/mdc-select/mdc-select.scss

```diff
@@ -132,12 +133,44 @@
   // stylelint-enable plugin/selector-bem-pattern
 }
 
-.mdc-select--disabled {
-  @include mdc-select-dd-arrow-svg-bg_($mdc-select-disabled-arrow-color);
+.mdc-select--outlined {
+  @include mdc-select-outline-color($mdc-select-outlined-idle-border);
+  @include mdc-select-hover-outline-color($mdc-select-outlined-hover-border);
+  @include mdc-select-focused-outline-color(primary);
+  @include mdc-floating-label-float-position($mdc-select-outlined-label-position-y);
+  @include mdc-floating-label-shake-animation(text-field-outlined);
+  @include mdc-select-outline-corner-radius_($mdc-select-border-radius);
 
-  &.mdc-select--box {
-    @include mdc-select-container-fill-color_($mdc-select-box-disabled-fill-color);
+  height: 56px;
+  border: none;
+  background-position: right 10px center;
+
+  @include mdc-rtl {
+    background-position: left 10px center;
+  }
+
+  .mdc-select__native-control {
+    @include mdc-rtl-reflexive-property(padding, $mdc-select-label-padding, $mdc-select-arrow-padding);
+
+    display: flex;
```

Comment:
idea...this looks exactly like the `@mixin mdc-text-field-outlined_` piece of sass. What if we put this as a mixin in the notched-outline sass?